devito/finite_differences/finite_difference.py

```diff
@@ -89,7 +99,10 @@ def second_derivative(expr, **kwargs):
     ind = [(dim + i * diff) for i in range(-int(order / 2),
                                            int(order / 2) + 1)]
 
-    coeffs = finite_diff_weights(2, ind, dim)[-1][-1]
+    if fd_weight_type is 'symbolic':
+        coeffs = symbolic_weights(f, 2, indices, dim)
```

Comment:
what is ``f`` here? it seems undefined, unless you're referring to the last ``f`` in the loop above, which would clearly be flawed